src/theme/BlogPostItem/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { DiscussionEmbed } from 'disqus-react'
+import BlogPostItem from '@theme-original/BlogPostItem';
+import { useBlogPost } from '@docusaurus/theme-common/internal'
```

Comment:
There is a problem with this import. We need to find a better way of getting post metadata that works with the current version of Docusaurus.